Assets/Scripts/Game/UserInterfaceWindows/DaggerfallSpellMakerWindow.cs

```diff
@@ -743,6 +743,22 @@ private void BuyButton_OnMouseClick(BaseScreenComponent sender, Vector2 position
             //   Implement costs and remove gold from player and block if user does not have enough gold (message 1702)
             //   Spells will be free (both gold and spell points) during build-out of effect system.
 
+            var moneyAvailable = GameManager.Instance.PlayerEntity.GoldPieces;
+            var cost = totalGoldCost;
```

Comment:
Probably no need to assign this to its own field here. You could just use

    if (moneyAvailable < totalGoldCost)
    ...
    GameManager.Instance.PlayerEntity.DeductGoldAmount(totalGoldCost);